[DOC] Tokenizer - Letter (#8498)
opensearch-project/documentation-website: _analyzers/tokenizers/letter.md

---
layout: default
title: Letter
parent: Tokenizers
nav_order: 60
---

# Letter tokenizer

> The `letter` tokenizer splits text into words on any non-letter characters. It works well for many European languages but struggles with some Asian languages where words aren't separated by spaces.

Reviewer: ```suggestion
The `letter` tokenizer splits text into words on any non-letter characters. It works well with many European languages but is less effective with some Asian languages in which words aren't separated by spaces.
```
